TUTORIAL.rst

```diff
@@ -0,0 +1,112 @@
+PDFmorph Tutorial
+#################
+
+Welcome! This will be a quick tutorial to accquaint users with PDFmorph
+and some of what it can do. 
+
+As we described in the README and installation instructions, please make
+sure that you are familiar with working with your command line terminal
+before using this application.
+
+Before you've started this tutorial, please ensure that you've installed
+all necessary software and dependencies.
+
+PDFmorph Example
+----------------
+
+	1. Open your Terminal or Command Prompt.
+
+	2. It it's not active already, activate your PDFmorph-equipped 
+	   conda environment by typing in ::
+	
+		source activate <pdfmorph_env>
+
+	   on Linux or ``activate <pdfmorph_env>`` on Windows.
+
+		* If you need to list your available conda environments,
+		  run the command ``conda info --envs`` or 
+		  ``conda env list``
+
+	3. Change directories with the ``cd`` command to a directory
+	   containing PDF files. For instance, I will use the file
+	   "ni_qmax25.cgr", found `here<https://github.com/diffpy/diffpy.pdfmorph/tree/master/tests/testdata>`_,
+	   so I will run ``cd ~/diffpy.pdfmorph/diffpy/pdfmorph/tests/testdata``
+
```

Comment:
The first thing to have them do is to type ``PDFmorph -h`` so they know how to access the help.

Also, at this point, encourage them to joing diffpy-users and let them know they can post questions about how to use the software (also bug reports) there.